src/hooks/useSwapEstimateBot.ts

```diff
@@ -21,7 +21,6 @@ type PriceData = {
 }
 
 const useSwapEstimateBot = () => {
-  const [isFetching, setFetching] = useState(false)
```

Comment:
isFetching is mostly needed for UI feedback, I don't think it need to be in a hook

Reply:
Also makes us a lot flexible with setting it compared to keeping it in a hook

Reply:
Sure, you can more it to utils.